kotlin-runtime/external-module-template/target.tmpl/generated-sources/ftl/{{ range .ExternalModules }}{{ push .Name . }}{{ end }}/{{ .Name | camel }}.kt

```diff
@@ -0,0 +1,34 @@
+{{- $moduleName := .Name -}}
+// Code generated by FTL. DO NOT EDIT.
+{{.Comments|comment -}}
+package ftl.{{.Name}}
+{{- $imports := (.|imports) -}}
+{{- if $imports}}
+{{range $import := $imports}}
+import {{$import}}
+{{- end}}{{end}}
```

Comment:
This is quite difficult to read. Let's introduce some whitespace here between the different semantic sections. The intent of using `{{- ` and `- }}`, is to allow the template to have meaningful whitespace while still generating clean output.

Reply:
updated! hopefully clearer chunks now